src/main/scripts/src/app/core/header/header.component.html

```diff
@@ -17,5 +17,17 @@
         {{workspaceService.organizationCode}}@{{workspaceService.projectCode}}</span></a>
   </div>
   <div class="notifications">0</div>
-  <div class="profile"></div>
+
+  <!-- User icon -->
+  <a #profile class="profile">
+
+    <div *ngIf="optionsVisible" class="arrow-up"></div>
+    <ul #options *ngIf="optionsVisible" [@animateHeight]="'in'">
+      <li><a href=""><i class="fa fa-cog" aria-hidden="true"></i>Settings</a></li>
```

Comment:
This link should point to `/user/settings`. See the examples of links in [other parts of the application](https://github.com/Lumeer/web-ui/blob/devel/src/main/scripts/src/app/core/header/header.component.html#L15).